cfgov/unprocessed/apps/analytics-gtm/js/bah-loan-estimate-listeners.js

```diff
@@ -1,68 +1,142 @@
-    let ele = $( this ),
-        tab = ele.closest( '.explain' ).find( '.active-tab' ),
-        tabText = tab.find( '.tab-label' ).text().trim();
-    delay(
-      function() {
-        let state = ele.attr( 'aria-pressed' ),
-            action = 'Expandable collapsed - ' + tabText,
-            label = $( '<p>' + ele.find( '.o-expandable_label' ).html() + '</p>' ),
-            text = '';
-
-        label.find( 'span' ).empty();
-        text = label.text().trim();
-
-        if ( state === 'true' ) {
-          action = 'Expandable expanded - ' + tabText;
-        }
-        track( 'OAH Loan Estimate Interaction', action, text );
-      }, 250 );
-  } );
-
-  $( '.image-map_overlay' ).click( function() {
-    let href = $( this ).attr( 'href' ),
-        text = $( this ).text().trim();
-    delay(
-      function() {
-        let action = 'Image Overlay click - expandable collapsed',
-            target = $( href );
-        if ( target.hasClass( 'o-expandable__expanded' ) ) {
-          action = 'Image Overlay click - expandable expanded';
-        }
-        track( 'OAH Loan Estimate Interaction', action, text );
-      }, 250 );
-  } );
-
-} )( $ );
+  }
+
+  /**
+   * @param {MouseEvent} event - Mouse event from the click.
+   */
+  function trackExpandableTargetsClick( event ) {
+    const elem = event.currentTarget;
+    const expandable = elem.parentNode;
+    const expandableID = expandable.id;
+    if ( isAnimatingExpandable( expandable ) ) { return; }
+    recordExpandableState( expandableID );
+
+    const tab = closest( elem, '.explain' ).querySelector( '.active-tab' );
+    const tabText = tab.querySelector( '.tab-label' ).textContent.trim();
+    let action = 'Expandable collapsed - ' + tabText;
+    const label = elem.querySelector( '.o-expandable_label' );
+    let text = '';
+    text = label.textContent.trim();
+    if ( expandableStates[expandableID] === true ) {
+      action = 'Expandable expanded - ' + tabText;
+    }
+    track( 'OAH Loan Estimate Interaction', action, text );
+  }
+
+  /**
+   * @param {MouseEvent} event - Mouse event from the click.
+   */
+  function trackImageMapOverlayClick( event ) {
+    const target = event.target;
+    const href = target.getAttribute( 'href' );
+    const text = target.textContent.trim();
+
+    let action = 'Image Overlay click - expandable collapsed';
+    const expandable = document.querySelector( href );
+    const expandableID = expandable.id;
+    if ( isAnimatingExpandable( expandable ) ) { return; }
+    recordExpandableState( expandableID );
+    if ( expandableStates[expandableID] === true ) {
+      action = 'Image Overlay click - expandable expanded';
+    }
+    track( 'OAH Loan Estimate Interaction', action, text );
+  }
+
+  const tabLinksDom = document.querySelectorAll( '.tab-link' );
+  addEventListenerToElems( tabLinksDom, 'click', trackTabLinkClick );
+
+  const formExplainerPageLinksDom =
+    document.querySelectorAll( '.form-explainer_page-link' );
```

Comment:
Why not just allow `addEventListenerToElems` to accept a selector or DOM element?

Reply:
I updated this to `addEventListenerToSelector`